tests/integration/test_data_cli.py

```diff
@@ -79,6 +81,96 @@ def test_data_search_quick_filter_success(invoke, item_types):
 
 
+@respx.mock
+@pytest.mark.asyncio
+@pytest.mark.parametrize("filter", ['{1:1}', '{"foo"}'])
+@pytest.mark.parametrize(
+    "item_types", ['PSScene', 'SkySatScene', ('PSScene', 'SkySatScene')])
+def test_data_search_create_filter_invalid_json(invoke, item_types, filter):
+    """Test for planet data search_create. Test with multiple item_types.
+    Test should fail as filter does not contain valid JSON."""
+    mock_resp = httpx.Response(HTTPStatus.OK,
+                               json={'features': [{
+                                   "key": "value"
+                               }]})
+    respx.post(TEST_SEARCHES_URL).return_value = mock_resp
+
+    name = "temp"
+
+    runner = CliRunner()
+    result = invoke(["search-create", name, item_types, filter], runner=runner)
+    assert result.exit_code == 2
+
+
+@respx.mock
+@pytest.mark.parametrize(
+    "item_types", ['PSScene', 'SkySatScene', ('PSScene', 'SkySatScene')])
+def test_data_search_create_filter_success(invoke, item_types):
+    """Test for planet data search_create. Test with multiple item_types.
+    Test should succeed as filter contains valid JSON."""
+    filter = {
+        "type": "DateRangeFilter",
+        "field_name": "acquired",
+        "config": {
+            "gt": "2019-12-31T00:00:00Z", "lte": "2020-01-31T00:00:00Z"
+        }
+    }
+
+    name = "temp"
+
+    mock_resp = httpx.Response(HTTPStatus.OK,
+                               json={'features': [{
+                                   "key": "value"
+                               }]})
+    respx.post(TEST_SEARCHES_URL).return_value = mock_resp
+
+    runner = CliRunner()
+    result = invoke(["search-create", name, item_types, json.dumps(filter)],
+                    runner=runner)
+
+    assert result.exit_code == 0
+    assert len(result.output.strip().split('\n')) == 1  # we have 1 feature
+
+
+@respx.mock
+def test_search_create_daily_email(invoke, search_result):
+    mock_resp = httpx.Response(HTTPStatus.OK, json=search_result)
+    respx.post(TEST_SEARCHES_URL).return_value = mock_resp
+
+    filter = {
+        "type": "DateRangeFilter",
+        "field_name": "acquired",
+        "config": {
+            "gt": "2019-12-31T00:00:00Z", "lte": "2020-01-31T00:00:00Z"
+        }
+    }
+
+    result = invoke([
+        'search-create',
+        'temp',
+        'SkySatScene',
+        json.dumps(filter),
+        '--daily_email'
+    ])
+
+    search_request = {
+        "name": "temp",
+        "filter": {
+            "type": "DateRangeFilter",
+            "field_name": "acquired",
+            "config": {
+                "gt": "2019-12-31T00:00:00Z", "lte": "2020-01-31T00:00:00Z"
+            }
+        },
+        "item_types": ["SkySatScene"],
+        "__daily_email_enabled": True
+    }
+    sent_request = json.loads(respx.calls.last.request.content)
+    assert result.exit_code == 0
+    assert sent_request == search_request
+    assert json.loads(result.output) == search_result
```

Comment:
excellent set of assertions. complete while only testing what is important. nicely done.